checks/check_extra7174

```diff
@@ -29,25 +29,25 @@ check7174(){
     for regx in ${REGIONS}; do
         LIST_OF_PROJECTS=$("${AWSCLI}" codebuild list-projects --region "${regx}" --query 'projects[*]' --output text 2>&1)
         if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "${LIST_OF_PROJECTS}"; then
-            textInfo "${regx}: Access Denied trying to list Codebuild projects"
+            textInfo "Access Denied trying to list Codebuild projects" "${regx}"
```

Comment:
Please, include `${regx}` too like you have it before.

Correct: `textInfo "${regx}: Access Denied trying to list Codebuild projects" "${regx}"`